src/frontend/csrs-portal/src/app/auth/auth-config.module.ts

```diff
@@ -1,34 +1,48 @@
+/* eslint-disable arrow-body-style */
+import { HttpClient } from '@angular/common/http';
 import { NgModule } from '@angular/core';
-import { AuthModule, LogLevel } from 'angular-auth-oidc-client';
+import { AuthModule, LogLevel, StsConfigHttpLoader, StsConfigLoader } from 'angular-auth-oidc-client';
+import { map } from 'rxjs/operators';
 
+export const httpLoaderFactory = (httpClient: HttpClient) => {
+  const config$ = httpClient.get<any>(`assets/oidc.json`).pipe(
+    map((customConfig: any) => {
+      return {
+        triggerAuthorizationResultEvent: true,
+        postLoginRoute: customConfig.postLoginRoute,
+        logLevel: LogLevel.Debug,
+        historyCleanupOff: true,
+        eagerLoadAuthWellKnownEndpoints: false,
+        authority: customConfig.authority,
+        redirectUrl: customConfig.redirectUrl,
+        postLogoutRedirectUri: customConfig.postLogoutRedirectUri,
+        clientId: customConfig.clientId,
+        scope: customConfig.scope,
+        responseType: customConfig.responseType,
```

Comment:
In our case the response type is always code - right Sergey?

Reply:
Nothing to modify here/we can load it from config if we want to. I am just curious.

Reply:
Agree, we should hard code everything that is standard and only save stuff in the external file that needs to change from the defaults.

Reply:
I hope that starting point is code. To be sure, we need more testing.